inst/include/common/information.hpp

```diff
@@ -200,11 +200,11 @@ class Information {
       FIMS_LOG << "Checking for available fleet age comp data objects."
                << std::endl;
       // set age composition data
-      if (f->observed_agecomp_data_id != -999) {
-        uint32_t agecomp_id =
-            static_cast<uint32_t>(f->observed_agecomp_data_id);
-        data_iterator it = this->data_objects.find(agecomp_id);
-        FIMS_LOG << "Input fleet age comp id = " << agecomp_id << "."
+      if (f->fleet_observed_agecomp_data_id_m != -999) {
+        uint32_t agecomp_id_m =
```

Comment:
[quibble] : add `info_` prior to `agecomp_id_m`?

Reply:
added info_ before agecomp_id_m